Rover/AP_MotorsUGV.h

```diff
@@ -84,6 +84,10 @@ class AP_MotorsUGV {
     void set_wingsail(float wingsail);
     float get_wingsail() const { return _wingsail; }
 
+    // set or get mast rotation input as a value from -100 to 100
+    void set_mast_rotation(float wingsail);
```

Comment:
this argument looks like it might be a cut-and-paste error

Reply:
You are absolutely correct. FIxed.